spec.bs

```diff
@@ -2907,6 +2907,7 @@ dictionary ReportWinBrowserSignals : ReportingBrowserSignals {
   DOMString buyerReportingId;
   unsigned short modelingSignals;
   unsigned long dataVersion;
+  DOMString kAnonStatus;

```

Comment:
The leading lowercase `k` is a Chromium style thing, right? We don't usually name web platform things with this style I think. Just usual camel case is preferred.

Reply:
Oops, of course!